projects/packages/publicize/src/class-publicize-script-data.php

```diff
@@ -67,8 +66,6 @@ public static function set_admin_script_data( $data ) {
 			$data['site']['plan']['features'] = Current_Plan::get_simple_site_specific_features();
 		}
 
-		$data['site']['wpcom']['blog_id'] = Manager::get_site_id( true );
-		$data['site']['suffix']           = ( new Status() )->get_site_suffix();
```

Comment:
Do these get added via the script data asset somewhere here instead?

Reply:
Previously we were assigning values to these properties only for public blogs, now I am assigning these values for all blogs and for this purpose I have updated the implemenation of [Script_Data::get_admin_script_data](https://github.com/Automattic/jetpack/pull/43085/files#diff-cfd2bf42a7227f57bedf03d2729228a7b1a30e47d48f0844d400e09d0fc46f47)